src/hooks/users.ts

```diff
@@ -19,7 +101,7 @@ const useConnectVenueUsers = () => {
       ? {
           collection: "users",
           where: ["enteredVenueIds", "array-contains", venueId],
-          storeAs: "users",
+          storeAs: "universeUsers",
```

Comment:
Since this is in `useConnectVenueUsers` it should probably `storeAs: venueUsers`. Alternatively, the name of this hook should be changed to something like `useConnectUniverseUsers`.